airflow/www/validators.py

```diff
@@ -76,3 +79,22 @@ def __call__(self, form, field):
             except JSONDecodeError as ex:
                 message = self.message or f"JSON Validation Error: {ex}"
                 raise ValidationError(message=field.gettext(message.format(field.data)))
+
+
+class ValidKey:
+    """
+    Validates values that will be used as keys
+
+    :param max_length:
+        The maximum length of the given key
+    """
+
+    def __init__(self, max_length=200):
+        self.max_length = max_length
+
+    def __call__(self, form, field):
+        if field.data:
+            try:
+                helpers.validate_key(field.data, self.max_length)
+            except AirflowException as e:
```

Comment:
Instead of AirflowException, should this catch everything? (`Exception`)

Reply:
Good call, done